rollup/conf/config.json

```diff
@@ -14,7 +14,6 @@
         "escalate_multiple_num": 2,
         "escalate_multiple_den": 1,
         "max_gas_price": 1000000000000,
-        "max_blob_gas_price": 10000000000000,
```

Comment:
Why is this removed?

Reply:
`LegacyTx`L2  sender (and future possible `DynamicFeeTx` L2 sender) do not need this config.